MyBlog.Services/Areas/Users/IUserService.cs

```diff
@@ -0,0 +1,16 @@
+﻿using MyBlog.Infrastructure.Entities;
+
+namespace MyBlog.Service.Areas.Users;
+
+public interface IUserService
+{
```

Comment:
also, you forget inject your dependency in the project.
In your case it`s program.cs file in the Api layer